README.md

```diff
@@ -148,6 +148,9 @@ The following settings are stored in `template_config.yml`.
                         'github'. To switch from Redmine to GitHub use the --migrate-github-issues
                         option.
 
+  keep_ci_update_for_latest_branches
+                        Total of branches for which CI updates are kept. The default is 5.
```

Comment:
Just since I am not 100% sure based on reading the code: `5` here means: 5 release branches of the format `[[:digit:]]\.[[:digit:]]*` plus the default branch (`main`), so six branches total?

Reply:
Actually, I can't see where the default branch is handled in this change, so I am guessing it is simply separate.

Reply:
yes, it would be 5 x.y + main,
the main branch is where the x.y branches are triggered (so it is already running on main)